js/src/utils.js

```diff
@@ -248,6 +271,37 @@ export function lastBytes(arr, num) {
   return safeSlice(arr, arr.length - num);
 }
 
+export function getErrMsg(e) {
+  switch (e.message) {
+    case 1:
+      return [false, errors.ERR_INVALID_CHAIN]
+    case 2:
```

Comment:
This needs to be rewritten depending on what we decide with `return` vs `throw` vs `[Boolean, any]`